recline-embed-button.tpl.php

```diff
@@ -11,6 +11,10 @@
 <div class="recline-embed">
   <a class="embed-link" href="#"><?php print t('Embed'); ?></a>
   <div class="embed-code-wrapper">
-    <textarea class="embed-code" style="height: 25px;" onclick="select()"><?php print $embed_url ?></textarea>
+    <textarea class="embed-code" style="height: 75px;width:100%;" onclick="select()"><?php print $embed_url ?></textarea>
```

Comment:
That creates a second box that only has the url -> not wrapped in the iframe tags.

Related with this commit https://github.com/NuCivic/recline/commit/2fcd18e5c8e26f63a03ede4bff7085b84b01a90d